go/mysql/constants.go

```diff
@@ -102,17 +102,17 @@ const (
 	// New 4.1 authentication. Always set, expected, never checked.
 	CapabilityClientSecureConnection = 1 << 15
 
-	// CLIENT_MULTI_STATEMENTS 1 << 16
+	// CapabilityClientMultiStatements is CLIENT_MULTI_STATEMENTS
 	// Can handle multiple statements per COM_QUERY and COM_STMT_PREPARE.
-	// Not yet supported.
+	CapabilityClientMultiStatements = 1 << 16
 
-	// CLIENT_MULTI_RESULTS 1 << 17
+	// CapabilityClientMultiResults is CLIENT_MULTI_RESULTS
 	// Can send multiple resultsets for COM_QUERY.
-	// Not yet supported.
+	CapabilityClientMultiResults = 1 << 17
 
-	// CLIENT_PS_MULTI_RESULTS 1 << 18
+	// CapabilityClientPsMultiResults is CLIENT_PS_MULTI_RESULTS
```

Comment:
Based on my comment in the other PR, I don't think we can support Execute, which is what this option is for. So, better to drop it.